git/homework.py

```diff
@@ -97,14 +106,21 @@ def is_word_in_text(word: str, text: str) -> bool:
         >>> False
 
     """
-    pass
+    if word.find(text):
```

Comment:
you do not need to use find. Look at the operator `in`.
And what if a word is not in the text. your function will return None instead False